lib/Parse/ParseDecl.cpp

```diff
@@ -7531,8 +7531,6 @@ static ParserStatus parseAccessorIntroducer(Parser &P,
       P.parseNewDeclAttribute(Attributes, /*AtLoc*/ {}, DAK_Consuming);
     } else if (P.Tok.isContextualKeyword("borrowing")) {
       P.parseNewDeclAttribute(Attributes, /*AtLoc*/ {}, DAK_Borrowing);
-    } else if (P.Tok.isContextualKeyword("transferring")) {
-      P.parseNewDeclAttribute(Attributes, /*AtLoc*/ {}, DAK_Transferring);
```

Comment:
@gottesmm You explicitly added this. Is there a plan to use `transferring` as a accessor modifier?

Reply:
No. I was just following what borrowing was doing.